projects/addon-doc/src/utils/color-conversion.ts

```diff
@@ -1,14 +1,11 @@
 export function rgbToHex(r: number, g: number, b: number): string {
-    return (
-        '#' +
-        [r, g, b]
-            .map(x => {
-                const hex = x.toString(16);
+    return `#${[r, g, b]
+        .map(x => {
+            const hex = x.toString(16);
 
-                return hex.length === 1 ? '0' + hex : hex;
-            })
-            .join('')
-    );
+            return hex.length === 1 ? `0${hex}` : hex;
```

Comment:
Do you think we can use `padStart` here? It seems to be supported by Safari 10

Reply:
fixed